ui_framework/components/icon/delete_icon.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { KuiIcon } from './kui_icon';
+
+export function DeleteIcon() {
+  return <KuiIcon className="fa-trash"/>;
```

Comment:
> What does the exports have to do with named imports? You'd still import them in the same way as before, this would just be an "implementation detail" within the ui_framework/components/icon folder.

Oops, sorry... I thought you showed imports.